auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -313,10 +315,12 @@ public static int getDefaultTheme() {
      * Signs the current user out, if one is signed in.
      *
      * @param activity The activity requesting the user be signed out.
-     * @return a task which, upon completion, signals that the user has been signed out
-     * ({@code result.isSuccess()}, or that the sign-out attempt failed unexpectedly
-     * ({@code !result.isSuccess()}).
+     * @return a task which, upon completion, signals that the user has been signed out ({@code
+     * result.isSuccess()}, or that the sign-out attempt failed unexpectedly ({@code
+     * !result.isSuccess()}).
+     * @deprecated use {@link #signOut(FragmentActivity)} instead
      */
+    @Deprecated
     public Task<Void> signOut(@NonNull Activity activity) {
```

Comment:
This makes me so happy: because our new `signOut` method uses `FragmentActivity` which extends `Activity`, people will be automatically upgraded. How's that for customer service! 😄